python/CMakeLists.txt

```diff
@@ -1,18 +1,25 @@
-set(SETUP_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in")
-set(SETUP_PY "${CMAKE_INSTALL_PREFIX}/python/setup.py")
-
-configure_file(${SETUP_PY_IN} ${SETUP_PY})
+if(NOT SKBUILD)
+  install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/MaterialX" DESTINATION "python" MESSAGE_NEVER)
+  install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/Scripts" DESTINATION "python" MESSAGE_NEVER)
+endif()
 
-install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/MaterialX" DESTINATION "python" MESSAGE_NEVER)
-install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/Scripts" DESTINATION "python" MESSAGE_NEVER)
+if(SKBUILD)
+  install(
+    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/Scripts/"
+    DESTINATION "${MATERIALX_PYTHON_FOLDER_NAME}/_scripts"
+    PATTERN "README.md" EXCLUDE
+  )
+endif()
 
 if(MATERIALX_PYTHON_OCIO_DIR)
   if(NOT EXISTS "${MATERIALX_PYTHON_OCIO_DIR}/config.ocio")
     message(WARNING "No file named config.ocio was found in the given OCIO directory.")
   endif()
-  install(DIRECTORY "${MATERIALX_PYTHON_OCIO_DIR}/" DESTINATION "python/MaterialX/config/" MESSAGE_NEVER)
+  install(DIRECTORY "${MATERIALX_PYTHON_OCIO_DIR}/" DESTINATION "${MATERIALX_PYTHON_FOLDER_NAME}/config/" MESSAGE_NEVER)
 endif()
 
-if(MATERIALX_INSTALL_PYTHON AND PYTHON_EXECUTABLE)
-  install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install clean --all)" MESSAGE_NEVER)
+if(MATERIALX_INSTALL_PYTHON AND PYTHON_EXECUTABLE AND NOT SKBUILD)
+  set(SETUP_PY "${CMAKE_INSTALL_PREFIX}/python/setup.py")
+  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in" "${SETUP_PY}")
+  install(CODE "execute_process(COMMAND ${PYTHON_EXECUTABLE} ${SETUP_PY} install clean --all)")
```

Comment:
I didn't have time to look into this, but I would like to get rid of the `setup.py` here. Maybe in a second PR.

Reply:
It's far from silly! TO be able to answer your question, I'll need more context on the history of this workflow and know more about how it's currently being used. Then I'll be able to see if we can improve the builds by removing the `setup.py` or at least improving it.

But I'll leave that as a separate task that can be dealt with later.